lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -236,45 +253,52 @@ def self.upload_binary(app_id, binary_path)
           request.headers["Authorization"] = "Bearer " + auth_token
         end
       rescue Faraday::ResourceNotFound
-        UI.crash!(ErrorMessage::INVALID_APP_ID)
+        UI.crash!(ErrorMessage::INVALID_APP_ID + app_id)
       rescue Errno::ENOENT
-        UI.crash!(ErrorMessage::APK_NOT_FOUND)
+        UI.crash!(ErrorMessage::APK_NOT_FOUND + binary_path)
       end
 
+      # Uploads the binary
+      #
+      # Returns the id of the release. Only happens on a successful release, on a fail release a messsage notifies the user.
```

Comment:
I think mentioning the return value in the failure case is more important than mentioning the message that's printed out. When we run out of retries, the return value should be nil, right?

It looks like we're also not handling this return value correctly when calling this method. Can you fix that too? I would also be fine with just crashing if that happens, in which case the return value does not matter or need to be handled.

Reply:
When it fails there should be an error message. I will update it so that it isn't a message but a user error that happens when the user runs out of tries.